src/components/Profile/History/index.tsx

```diff
@@ -66,6 +66,7 @@ export default function HistoryPage({
   const [refetchJobs, setRefetchJobs] = useState(false)
   const [isLoadingJobs, setIsLoadingJobs] = useState(false)
   const [jobs, setJobs] = useState<ComputeJobMetaData[]>([])
+  const [tabIndex, setTabIndex] = useState(0)
```

Comment:
suggestion: see below, we may want to initialize based on query param

Reply:
Implemented in https://github.com/deltaDAO/mvg-portal/pull/416